src/index.html

```diff
@@ -11,7 +11,119 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <fieldset class="otstup">
+        <legend>Personal information</legend>
+        <div class="formfield otstup2">
+          <label for="surname">Surname:</label>
+          <input type="text" name="surname"
+        autocomplete="off"
+        id="surname">
```

Comment:
```suggestion
          <input 
            type="text"
            name="surname"
            autocomplete="off"
            id="surname"
          >
```
Форматируй теги таким образом, если у них много атрибутов